### DISCUSSION TITLE
Does Squirrel.Windows support .net8 ?

### DISCUSSION THREAD

**Discussion by iamthefaker** [Squirrel/Squirrel.Windows#1881] [Ideas]
Does Squirrel.Windows support .Net8 ?
Is there anything extra that I need to know to get it to work with a .Net8 app?

**Comment by caesay** [comment]
Not properly, it does not. There are issues with SingleFile bundles, detecting SquirrelAware apps, there's no support for bootstrapping net8 at install time, and possibly other issues. There have been some PR's trying to patch Squirrel to work with .NET over the years but they have not been merged. Even if they had been merged, the last official release of Squirrel was in 2020 so it wouldn't be much help.

Here's a couple related issues
- https://github.com/Squirrel/Squirrel.Windows/pull/1554
- https://github.com/Squirrel/Squirrel.Windows/pull/1692

**Reply by iamthefaker** [reply]
Hope it is ok to ask this here, but is there a "reason" why Squirrel.Windows has lost momentum? ... Has it been superseded by something?

**Reply by anaisbetts** [reply]
It's lost momentum because no one is paid to work on it full-time any more. I'm semi-retired, and from a moral perspective, it feels Not Great to me to generate business value for hundreds of companies who are making money off of this software, in exchange for nothing

If e.g. GitHub wanted to hire me on a contracting basis to maintain Squirrel on a part-time basis I would definitely consider it, but as it stands, the idea of working for $0 for BigCorps is gross to me!

**Reply by legistek** [reply]
I can't blame you, but I'm astonished there wouldn't be sufficient demand to be able to charge something for this. All of Microsoft's installer options are garbage. What's left? InstallShield? No thank you. I'd gladly pay $100 or more a year for this to be properly supported and updated. Hell, the cost of a damned signing certificate is several times that.

**Comment by Light407** [comment]
Has there been any update regarding this?

**Reply by caesay** [reply]
No, Squirrel.Windows has still not had any updates since 2020 and there are many breaking bugs here. There are alternatives / continuations you can go looking for.
